app/invocation/invocation_action_card.tsx

```diff
@@ -262,9 +266,17 @@ export default class InvocationActionCardComponent extends React.Component<Props
       // If we have the executeResponseDigest in the URL, we can skip the
       // execution table lookup.
       const executeResponseDigest = parseActionDigest(executeResponseDigestParam);
+      if (!executeResponseDigest) {
+        alert_service.error("Missing execute response digest in URL");
```

Comment:
sorry I missed this earlier. It should say "Invalid", not "Missing", because we checked whether it's non-empty above.
```suggestion
        alert_service.error("Invalid execute response digest in URL");
```